WordPressAuthenticator/OAuth/ProofKeyForCodeExchange.swift

```diff
@@ -0,0 +1,41 @@
+// https://developers.google.com/identity/protocols/oauth2/native-app#step1-code-verifier
+//
+// FIXME: follow spec!
+//
+// A code_verifier is a high-entropy cryptographic random string using the unreserved
+// characters [A-Z] / [a-z] / [0-9] / "-" / "." / "_" / "~", with a minimum length of 43
+// characters and a maximum length of 128 characters.
+//
+// The code verifier should have enough entropy to make it impractical to guess the value.
+struct ProofKeyForCodeExchange {
+
+    enum Mode {
```

Comment:
I'm not good at naming things, but what do you think about renaming this to `Method` and the `var method` to something like `var googleOAuthParameterName`?

Reply:
Thank you for raising this. `Method` would definitely be a better name, as that's what [the spec](https://www.rfc-editor.org/rfc/rfc7636#page-17) uses 👍 

Addressed in 8abbb29ce3a4e795bcbb01890fb1826a7623e7f6.

As for the computed variable name, I went for `urlQueryParameterValue` to try and make it clearer where to use it.